src/redux/ensRegistration.ts

```diff
@@ -200,6 +218,32 @@ export const updateRegistrationDuration = (
   });
 };
 
+export const removeExpiredRegistrations = (
+  accountAddress: EthereumAddress
```

Comment:
same thing here for comment below:

seems like the `accountAddress` from `useAccountSettings` is always passed in - any reason this isn't just picked up from `getState` `settings` instead of being passed in as a param?

Reply:
same for the other functions in this file that are having `accountAddress` passed in (if only source is actually `useAccountSettings` and not some other account address)